src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -363,6 +364,51 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
     confirmedQuote,
   })
 
+  // Estimates *pool asset* outbound fees only for now
+  // TODO(gomes) - For the sake of simplicity, RUNE is not included in the estimation, we assume outbound fees are free which they aren't
+  // But once I get this working, I'll add the RUNE fees estimation, assuming we can get the rotating addresses from the pool module
+  // and estimations works from them
+  const estimateOutboundFeesArgs = useMemo(() => {
+    if (opportunityType === AsymSide.Rune) return undefined
+    if (!assetId || !wallet || !poolAsset || !poolAssetAccountAddress) return undefined
+    if (!inboundAddressesData) return undefined
+
+    const amountCryptoPrecision = actualAssetWithdrawAmountCryptoPrecision
+
+    return {
+      amountCryptoPrecision,
+      assetId: poolAsset.assetId,
+      to: poolAssetAccountAddress,
+      sendMax: false,
+      // Router holds tokens, inbound address holds native asset
```

Comment:
Another day another weirdness at protocol-level, gotta do what you gotta do to make things work